lightning/src/ln/blinded_payment_tests.rs

```diff
@@ -2160,7 +2160,7 @@ fn do_test_trampoline_single_hop_receive(success: bool) {
 		}
 		{
 			let payment_failed_conditions = PaymentFailedConditions::new()
-				.expected_htlc_error_data(INVALID_ONION_BLINDING, &[0; 0]);
+				.expected_htlc_error_data(0x4000 | 22, &[0; 0]);
```

Comment:
wondering if it makes sense to make a constant with the value `0x4000 | 22` that helps the reader to understand what this value is?

Reply:
See follow up PR #3601 :D

Reply:
brilliant 🥇